portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,14 +27,22 @@
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.FetchOptions;
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.files.FileServicePb.FileContentType.ContentType;
 import com.google.gson.Gson;
 
 /** Servlet that returns some example content.*/
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private static final String JSON_CONTENT_TYPE = "application/json;";
```

Comment:
nit: when naming constants like this it's often helpful to order the various name components from least specific to most specific. ie:

CONTENT_TYPE_JSON instead of JSON_CONTENT_TYPE
KIND_COMMENT instead of COMMENT_KIND
etc...

the motivation is that it's easier for readers to see groupings and understand context. for example:

KIND_COMMENT = ...
KIND_ABC = ...
KIND_Z = ...

it's easy for the reader to see 1) that these are related in the same category 2) what the distinction between each is (since the text will naturally line up more nicely). vs:

COMMENT_KIND = ...
ABC_KIND = ...
Z_KIND = ...